jenkins/spark-premerge-build.sh

```diff
@@ -95,25 +90,33 @@ rapids_shuffle_smoke_test() {
     $SPARK_HOME/sbin/stop-master.sh
 }
 
-unit_test() {
-    echo "Run unit testings..."
-    # Run the unit tests for other Spark versions but dont run full python integration tests
-    # NOT ALL TESTS NEEDED FOR PREMERGE
-    # Just test one 3.0.X version (base version covers this) and one 3.1.X version.
-    # All others shims test should be covered in nightly pipelines
-    # Disabled until Spark 3.2 source incompatibility fixed, see https://github.com/NVIDIA/spark-rapids/issues/2052
-    #env -u SPARK_HOME mvn -U -B $MVN_URM_MIRROR -Pspark320tests,snapshot-shims test -Dpytest.TEST_TAGS='' -Dcuda.version=$CUDA_CLASSIFIER
-    env -u SPARK_HOME mvn -U -B $MVN_URM_MIRROR -Pspark303tests,snapshot-shims test -Dpytest.TEST_TAGS='' -Dcuda.version=$CUDA_CLASSIFIER
-    env -u SPARK_HOME mvn -U -B $MVN_URM_MIRROR -Pspark304tests,snapshot-shims test -Dpytest.TEST_TAGS='' -Dcuda.version=$CUDA_CLASSIFIER
-    env -u SPARK_HOME mvn -U -B $MVN_URM_MIRROR -Pspark312tests,snapshot-shims test -Dpytest.TEST_TAGS='' -Dcuda.version=$CUDA_CLASSIFIER
-    env -u SPARK_HOME mvn -U -B $MVN_URM_MIRROR -Pspark313tests,snapshot-shims test -Dpytest.TEST_TAGS='' -Dcuda.version=$CUDA_CLASSIFIER
+integration_test() {
+    echo "Run integration testings..."
+    mvn -U -B $MVN_URM_MIRROR -pl 'integration_tests' -am clean verify -DskipTests=true -Dpytest.TEST_TAGS="not slow_test" \
```

Comment:
I think `-DskipTests=true` will skip IT too reg: https://github.com/NVIDIA/spark-rapids/blob/branch-21.10/integration_tests/run_pyspark_from_build.sh#L20

'-P!snapshot-shims' could add this to skip snapshot shims build

Reply:
there is not too much ENVs required for pre-merge tests run, all others could be kept as default
https://github.com/NVIDIA/spark-rapids/blob/branch-21.10/integration_tests/pom.xml#L167-L176

Reply:
yes, that's right. Refined now